### PR TITLE
Project setup with create-react-app

### PR DESCRIPTION
**Pull request by codeAesthetic**
The project was set up with the help of the **[create-react-app](https://www.npmjs.com/package/create-react-app)** library.
command used for creating typescript template ``create-react-app my-app --template typescript``

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.8",
+    "@testing-library/react": "^11.2.2",
+    "@testing-library/user-event": "^12.6.0",
+    "@types/jest": "^26.0.19",
+    "@types/node": "^12.19.11",
+    "@types/react": "^16.14.2",
+    "@types/react-dom": "^16.9.10",
```
Comment (chirgjn) [inline package.json:12]: Were these added by CRA?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-router-dom": "^5.2.0",
+    "react-scripts": "4.0.1",
+    "styled-components": "^5.3.0",
+    "typescript": "^4.1.3",
```
Comment (chirgjn) [inline package.json:18]: Were these added by CRA?

**Review by chirgjn — CHANGES_REQUESTED** [review]
Please only include relevant changes to the PR

Please also mention the commands used to setup the project in PR description

File: package.json (hunk 2 above)
Comment (codeAesthetic) [reply package.json:18]: only styled components was added later.

File: README.md [hunk 3]
```diff
@@ -1 +1,6 @@
 # Juntos Frontend
+
+### How to use Project.
+
+- run `yarn install` or `npm install` to install all required project dependencies.
+- run `yarn start` or `npm start` to run project.
```
[the remark this answers is not in the document]
Comment (codeAesthetic) [reply README.md:6]: NO, I added this.

File: package.json (hunk 1 above)
Comment (codeAesthetic) [reply package.json:12]: yes

**Comment by chirgjn** [comment]
Please remove the unrelated changes from this PR

File: README.md [hunk 4]
```diff
@@ -1 +1,6 @@
 # Juntos Frontend
+
+### How to use Project.
```
Comment (chirgjn) [inline README.md:3]: Why are we using a level 3 heading, instead of level 2, will there be a parent section?

**Review by chirgjn — CHANGES_REQUESTED** [review]
- Remove `.DS_Store`
- This has both `yarn.lock` and `packag-lock.json`, let's document the recommended package manager to use and remove the extra file
- Document that the App is built using `create-react-app` in the readme

File: README.md (hunk 4 above)
Comment (codeAesthetic) [reply README.md]: gave it heading 2. No there wont be any such parent section

File: README.md [hunk 5]
```diff
@@ -1 +1,8 @@
 # Juntos Frontend
+
+The project was set up with the help of the **[create-react-app](https://www.npmjs.com/package/create-react-app)** library.
+command used for creating typescript template ``create-react-app my-app --template typescript``
+## How to use Project.
+
```
Comment (chirgjn) [inline README.md:6]: extra line?

File: README.md [hunk 6]
```diff
@@ -1 +1,8 @@
 # Juntos Frontend
+
+The project was set up with the help of the **[create-react-app](https://www.npmjs.com/package/create-react-app)** library.
+command used for creating typescript template ``create-react-app my-app --template typescript``
```
Comment (chirgjn) [inline README.md:4]: Can you add a blank line below to separate the sections

**Review by chirgjn — CHANGES_REQUESTED** [review]
> - Remove `.DS_Store`

File: README.md (hunk 5 above)
Comment (codeAesthetic) [reply README.md:6]: doesn't matter both yield same result.

**Review by chirgjn — APPROVED** [review]
LGTM

**Pull request merged**